libs/application/templates/social-insurance-administration/household-supplement/src/fields/Review/review-groups/Attachments.tsx

```diff
@@ -0,0 +1,40 @@
+import { Label, ReviewGroup } from '@island.is/application/ui-components'
+import { Box, TopicCard } from '@island.is/island-ui/core'
+import { ReviewGroupProps } from './props'
+import { useLocale } from '@island.is/localization'
+import { getAttachments } from '../../../lib/householdSupplementUtils'
+
+export const Attachments = ({ application }: ReviewGroupProps) => {
+  const { formatMessage } = useLocale()
+
+  const attachments = getAttachments(application)
+
+  return (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
```

Comment:
You can remove the fragment and the brackets and the commented out line to avoid disabling eslint. We try to avoid disabling eslint rules unless its absolutely necessary.